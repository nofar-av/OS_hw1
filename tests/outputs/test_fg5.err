/bin/bash: Z^: command not found
/bin/bash: ^j^o^b^s^: command not found
/bin/bash: ^q^u^i^t^: command not found
