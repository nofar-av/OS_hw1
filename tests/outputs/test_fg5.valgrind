==27129== Memcheck, a memory error detector
==27129== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27129== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27129== Command: /home/omer/OS_wet1/smash
==27129== Parent PID: 27126
==27129== 
==27129== 
==27129== FILE DESCRIPTORS: 4 open at exit.
==27129== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_fg5.valgrind
==27129==    <inherited from parent>
==27129== 
==27129== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_fg5.err
==27129==    <inherited from parent>
==27129== 
==27129== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_fg5.out
==27129==    <inherited from parent>
==27129== 
==27129== Open file descriptor 0:
==27129==    <inherited from parent>
==27129== 
==27129== 
==27129== HEAP SUMMARY:
==27129==     in use at exit: 0 bytes in 0 blocks
==27129==   total heap usage: 73 allocs, 73 frees, 84,049 bytes allocated
==27129== 
==27129== All heap blocks were freed -- no leaks are possible
==27129== 
==27129== For counts of detected and suppressed errors, rerun with: -v
==27129== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
