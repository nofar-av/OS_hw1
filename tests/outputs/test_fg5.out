smash> submitted by Eran
smash> smash> smash> smash> sleep 200& : 5559
smash: got ctrl-Z
smash: process 5559 was stopped
smash> [1] sleep 100& : 5558 1 secs
[2] sleep 200& : 5559 0 secs (stopped)
[3] sleep 300& : 5560 1 secs
smash> 